internal/controller/drpolicy_controller.go

```diff
@@ -178,10 +177,36 @@ func (r *DRPolicyReconciler) initiateDRPolicyMetrics(drpolicy *ramen.DRPolicy, d
 	return nil
 }
 
+func (r *DRPolicyReconciler) getDRClusterDetails(ctx context.Context) (*ramen.DRClusterList, map[string]string, error) {
+	drClusters := &ramen.DRClusterList{}
+	if err := r.Client.List(ctx, drClusters); err != nil {
+		return nil, nil, fmt.Errorf("drclusters list: %w", err)
+	}
+
+	drClusterIDToName := map[string]string{}
```

Comment:
would it make sense to rename this map to `drClusterIDsToNames` or `drClusterIDNameMap`. This `drClusterIDToName` looked like it holds a single value.